src/EkoPlayer.js

```diff
@@ -212,6 +213,16 @@ class EkoPlayer {
         this.ekoEmbed.once(eventName, callback);
     }
 
+    /**
+     * Will dispose EkoPlayer instance.
+     *
+     * @memberof EkoPlayer
+     */
+    dispose() {
+        this.iframe.remove();
```

Comment:
The iframe is removed from the DOM, but not disposed of, so for instance, audio might continue playing, memory may not be released etc.

Consider changing implementation to:

```js
dispose() {
    if (this.iframe && typeof this.iframe.remove === 'function') {
        this.iframe.remove();
    }
    if (this.ekoEmbed && typeof this.ekoEmbed.dispose === 'function') {
        this.ekoEmbed.dispose();
    }
    this.iframe = null;
    this.ekoEmbed = null;
}
```